### ISSUE TITLE
Moving the Alan XSL-FOP Template to an Independent Repo

### ISSUE THREAD

**Issue by tajmone** [alan-if/alan-docs#44]
@thoni56 I was thinking of moving the [`xsl-fopub/`][xsl-fopub] folder to a repository of its own and the include it back as a Git submodule.

The reason for this is that other Alan related projects like Anssi's StdLib (and, eventually, also my Alan Italian project) will need the AsciiDoc fopub template to build their documentation in PDF. 

Since the StdLib repo dynamically compiles source adventures and generates transcripts to import into the documentation sources, they need to keep the documentation together with the library sources. So, moving the StdLib documentation to this repository is not a viable option for practical reasons (besides, keeping everything together is better for maintainance purposes).

Moving the Alan AsciiDoc PDF template to a standalone repository would allow any Alan related project to benefit from it without having to manually import it, which would also prevent having multiple copies of the template. The obvious benefits would be that any updates to the template repository could be automatically integrated by moving the Git submodule pointer.

Would that be OK with you?

If yes, I'd create the new repository under the Alan-IF organization.

[xsl-fopub]: https://github.com/alan-if/alan-docs/tree/master/_assets/xsl-fopub

**Comment by thoni56** [comment]
That looks like a valid refactoring, so please, go ahead.

**Comment by tajmone** [comment]
Great! The new repository will be located here:

- https://github.com/alan-if/alan-xsl-fopub

**Comment by tajmone** [comment]
Ok @thoni56 , the new repository was successfully created and I've enforced the new submodule system on the Alan Docs project.

I haven't really worked often with submodules before, so I hope everything is fine.

All tests on `master` branch seem fine so far, and I've merged `master` on all the dev branches too (except `handle_elisions`, which was already merged into `beta7-prep` and is being kept only for safety) to make sure they don't mess up the submodule when switching branches. Again, everything seems fine on those branches too.

Let me know if everything works fine with you too.
